CMakeLists.txt

```diff
@@ -835,6 +835,18 @@ option(TENZIR_ENABLE_PYTHON_BINDINGS
        "Build Python bindings (requires poetry)" ON)
 add_feature_info("TENZIR_ENABLE_PYTHON_BINDINGS" TENZIR_ENABLE_PYTHON_BINDINGS
                  "build Python bindings (requires poetry).")
+
+set(TENZIR_ENABLE_BUNDLED_UV_default "ON")
+if (DEFINED ENV{TENZIR_ENABLE_BUNDLED_UV})
+  set(TENZIR_ENABLE_BUNDLED_UV_default $ENV{TENZIR_ENABLE_BUNDLED_UV})
+endif ()
+
+cmake_dependent_option(
+  TENZIR_ENABLE_BUNDLED_UV "Install a statically linked copy of uv" "${TENZIR_ENABLE_BUNDLED_UV_default}"
+  "TENZIR_ENABLE_PYTHON_BINDINGS" OFF)
+add_feature_info("TENZIR_ENABLE_BUNDLED_UV" TENZIR_ENABLE_BUNDLED_UV
+                 "install a statically linked copy of uv.")
```

Comment:
The new dependency needs to be added so the SBOM file and to the dependency table in the instructions to build from source on docs.tenzir.com.

Reply:
I ran the SBOM update tool, since the `uv-binary` is also packaged in our `nix` setup now.